neurogym/envs/annubes.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+
+import neurogym as ngym
+from neurogym import TrialEnv
+
+
+class AnnubesEnv(TrialEnv):
+    def __init__(
+        self,
+        session=None,
+        stim_intensities=None,
+        stim_time=1000,
+        decision_time=100,
+        catch_prob=0.5,
+        fix_intensity=0,
+        fix_time=500,
+        dt=100,
+        tau=100,
+        n_outputs=2,
+        output_behavior=None,
+        noise_std=0.01,
+        random_seed=None,
+    ):
+        if output_behavior is None:
+            output_behavior = [0, 1]
+        if stim_intensities is None:
+            stim_intensities = [0.8, 0.9, 1.0]
+        if session is None:
+            session = {"v": 0.5, "a": 0.5}
+        super().__init__(dt=dt)
+
+        self.session = session
+        self.stim_intensities = stim_intensities
+        self.stim_time = stim_time
+        self.decision_time = decision_time
+        self.catch_prob = catch_prob
+        self.fix_intensity = fix_intensity
+        self.fix_time = fix_time
+        self.dt = dt
+        self.tau = tau
+        self.n_outputs = n_outputs
+        self.output_behavior = output_behavior
+        self.noise_std = noise_std
+        self.random_seed = random_seed
```

Comment:
I think this should be discussed in a dedicated issue, together with [the configuration issue](https://github.com/ANNUBS/annubes/issues/36). In the previous annubes, I was not entirely convinced about the dataclasses, so this could be a good opportunity to reconsider that approach. Revisiting it could also help address the issue of having too many parameters, whether by using a combination of dataclasses and a configuration file, or by considering alternative solutions.

For now, I suggest we maintain the current type and structure of the class as closely aligned as possible with the other environment classes, until we revisit the points raised in issue #22. Feel free to open a new issue if you think is better to keep track of this.